src/ScrollPage.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import ScrollTop from './ScrollTop'
+require('./main-sass.scss')
+
+
+export default class ScrollPage extends React.Component {
+    render() {
+        const pageBody = {
+            height : "1000px"
+        }
+        return (
+            <div style={pageBody}>
+                text text text
+                <ScrollTop name="arrow-up" position={{top: "auto",left: "auto", right: 20, bottom: 20}} backgroundColor="blue" scrollOffset={30} />
```

Comment:
Please create object inside render like
```javascript
let position = {
top: "auto",
left: "auto", 
right: 20,
bottom: 20
}

```  
And pass it as props 

```javascript

 <ScrollTop name="arrow-up" position={position} backgroundColor="blue" scrollOffset={30} />

``` 
For better readability.